src/CreateEntityMappingView/CreateEntityMappingView.js

```diff
@@ -391,12 +396,14 @@ class CreateEntityMappingView extends ConnectionComponent {
 		}
 		return (
 			<Container>
+				<Paper style={{paddingLeft: 20, paddingRight: 20, marginTop: 40, paddingTop: 20}} zDepth={2}>
 				<div>
 					{this.renderSelectTargetStatus()}
 					{this.renderSelectTypes(initialData)}
 					{initialData.attributeFields}
 					{CreateEntityMappingView.renderButtons(submitCallback)}
```

Comment:
The buttons are not centered are they? Seems like the submit button is too far on the right.